### ISSUE TITLE
Feil: TextArea med teller feiler på axe-test

### ISSUE THREAD

**Issue by wkillerud** [fremtind/jokul#2780]
## Feilbeskrivelse

Ved test med `jest-axe@5.0.1` og nyeste `TextArea` med `counter` så feiler testen.

## Steg for å gjenskape

- Gå til TextArea.test.tsx
- Legg til propen `counter={{ maxLength: 200 }}`
- Kjør testene og se at du får en feil

```
Summary of all failing tests
 FAIL  ../packages/text-input-react/src/TextArea.test.tsx
  ● a11y › text-area should be a11y compliant

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('.jkl-text-area__counter') to have no violations:

    <div class="jkl-text-area__counter" aria-live="polite" aria-label="0 av 200 tegn brukt"><div class="jkl-text-area__counter-count" aria-hidden="true">0&nbsp;/&nbsp;200</div><div class="jkl-text-area__counter-progress" aria-hidden="true" style="--progress-width: 100%;"></div></div>

    Received:

    "Elements must only use allowed ARIA attributes (aria-allowed-attr)"

    Fix all of the following:
      ARIA attribute cannot be used, add a role attribute or use a different element: aria-label

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI

      32 |         const results = await axe(container);
      33 |
    > 34 |         expect(results).toHaveNoViolations();
         |                         ^
      35 |     });
      36 | });
      37 |
```

## Forventet oppførsel

Testen bør kjøre uten feil
